src/main/kotlin/net/maizegenetics/phgv2/api/HaplotypeGraph.kt

```diff
@@ -208,7 +213,7 @@ class HaplotypeGraph(hvcfFiles: List<String>) {
         myLogger.info("getSampleNamesALTHeaders: ${hvcfFiles.size} hvcf files")
 
         // Step 1: Get sample names and ALT headers
-        val sampleChannel = Channel<Deferred<Pair<List<String>, Map<String, AltHeaderMetaData>>>>(25)
+        val sampleChannel = Channel<Deferred<Triple<List<String>, Map<String, AltHeaderMetaData>, List<VCFContigHeaderLine>>>>(25)
```

Comment:
Consider turning this Triple into a specific data class.  That way you have named parameters and it is clear what information goes with which variable.